### DISCUSSION TITLE
人工智慧淪為寫作業神器，紐約公立學校禁用 ChatGPT

### DISCUSSION THREAD

**Discussion by Valentinaa0214** [1101-GS3073-C/Python-Community-GS3073-C#643] [General]
不知道大家對這個議題的看法是甚麼，
老實說我自己有時也會用ChatGPT來「點綴」一下作業，
但這麼方便的工具的確很有可能被大眾過度濫用、過度依賴，
導致失去獨立思考、表達自己看法的能力，甚麼都跑去問機器人。
(https://technews.tw/2023/01/06/nyc-education-department-blocks-chatgpt-on-school-devices-and-networks/)

**Comment by boson13579** [comment]
同意你的看法。雖然ChatGPT對於生成文章真的好用，但也不能過於依賴。學生在做報告和撰寫作業時還是最好通過自己的思考、組織整理一下自己得出的所有資訊、想出合適的詞語之後再寫下想法會比較好。
